packages/db/src/async_operations.ts

```diff
@@ -0,0 +1,33 @@
+import { toSnake } from 'snake-camel';
+
+interface CamelCaseAsyncOperation {
+  id: string
+  description: string
+  operationType: string
+  status: string
+  output?: object
+  taskArn?: string
+  createdAt?: Date
+  updatedAt?: Date
+}
+
+interface SnakeCaseAsyncOperation {
+  id: string
+  description: string
+  operation_type: string
+  status: string
+  output?: object
+  task_arn?: string
+  created_at?: Date
+  updated_at?: Date
+}
+
+export const translateAsyncOperationToSnakeCase = (
```

Comment:
Hmm. I know that you and @Jkovarik had discussed this, but I'm not sure I totally understand the utility of this helper if it's only used one place in the async operations package

Reply:
We had a conversation during the PR review - I think an approach like the one in #1992 works, but further conversation may be warranted.    In either case I agree re: the name change